docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -277,16 +277,45 @@ in the partition::
     DELETE FROM iceberg.testdb.customer_orders
     WHERE country = 'US' AND customer = 'Freds Foods'
 
-Rolling back to a previous snapshot
------------------------------------
+
+
+Snapshots
+---------
 
 Iceberg supports a "snapshot" model of data, where table snapshots are
 identified by an snapshot IDs.
 
 The connector provides a system snapshots table for each Iceberg table.  Snapshots are
-identified by BIGINT snapshot IDs.  You can find the latest snapshot ID for table
+identified by BIGINT snapshot IDs.  You can find the snapshot IDs for table
 ``customer_orders`` by running the following command::
 
+    SELECT snapshot_id FROM iceberg.testdb."customer_orders$snapshots" ORDER BY committed_at
```

Comment:
may be keep `DESC` as it's a little bit easier to see when someone is playing around